src/main/java/de/zalando/aruha/nakadi/service/subscription/StreamingContext.java

```diff
@@ -160,7 +160,7 @@ private void rebalance() {
             clientListChanges.refresh();
             zkClient.runLocked(() -> {
                 final Partition[] changeset = rebalancer.apply(zkClient.listSessions(), zkClient.listPartitions());
-                if (changeset != null && changeset.length > 0) {
```

Comment:
We have apache commons in our dependencies why not to use it? ArrayUtils.isNotEmpty()